apps/explorer/src/components/common/ProgressBar/index.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react'
 
+import { PercentDisplay } from '@cowprotocol/ui/pure/PercentDisplay'
```

Comment:
@alfetopito @anxolin please pay attention here.
I had to do that because importing from `@cowprotocol/ui` cause the errors: https://vercel.com/cowswap/explorer-dev/7TFVkMzbAvMaGLJ1oF9rknMVrp4n

Mainly, it's because when we import from `index.ts` we get all entities that exported from the index. But in this case I only need one component.

@alfetopito We could solve the similar issue with importing `useFeatureFlags` in a similar way.

Reply:
The fix: https://github.com/cowprotocol/cowswap/pull/4482/commits/49694703af27493174aa137e1790bc3e048e5396